app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,68 @@
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_recipe, only: %i[show edit destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = current_user.recipes.all
+  end
+
+  # GET /recipes/1 or /recipes/1.json
+  # def show
+  #   @recipe = Recipe.includes(:recipe_foods).find(params[id])
+  # end
```

Comment:
Thank you I will remove them

Reply:
**OPTIONAL**
- [ ] This change has not yet been implemented. As discussed by the previous reviewer please remove all the commented code in your project. This will keep the application clean

Reply:
Thank you, I removed the comments